src/System.Net.Http/tests/FunctionalTests/CancellationTest.cs

```diff
@@ -28,7 +28,7 @@ public CancellationTest(ITestOutputHelper output)
         [InlineData(false, true)]
         [InlineData(true, false)]
         [InlineData(true, true)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19038")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "uap: dotnet/corefx #20010, netfx: dotnet/corefx #19038")]
```

Comment:
@safern added more options, but we didn't communicate broadly:

``` 
ActiveIssue (int issue, TargetFrameworkMonikers framework)
 
ActiveIssue (int issue, TestPlatforms platform)
 
ActiveIssue (int issue, TestPlatforms platform = TestPlatforms.Any, TargetFrameworkMonikers framework = (TargetFrameworkMonikers)0)
 
This allows us to do the following things:
1.	Pass in the issue number itself and disable the test for all test platforms and all target framework monikers. [ActiveIssue(1232)]
2.	Call it with issue number and test platform, and it will disable the test for that specific platform and ALL target framework monikers. [ActiveIssue(1234, TestPlatforms.Windows)]
3.	Call it with issue number and target framework monikers, and it will disable the test for that specific target framework and ALL test platforms. [ActiveIssue(1234, TargetFrameworkMonikers.Netcoreapp)]
4.	Call it with issue number, test platform and target framework moniker, this will disable the test for that specific target framework ONLY on the specified platform. [ActiveIssue(2343, TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp)] – will disable the test on Windows only when running in netcoreapp. 
 
So this will give us the flexibility to disable a test with an active bug only in the flavor that it is actually failing to get more repros if so in different flavors and to continue having the test running in other target frameworks.
```